inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfanno/model/PdfExtractFile.java

```diff
@@ -42,22 +45,94 @@
     private String stringContent;
 
     /**
-     * Contains position mapping for a character between PDFExtract string
+     * Contains position mapping for characters between PDFExtract string
      * including and excluding Draw Operations.
      */
     private BidiMap<Integer, Integer> stringPositionMap;
 
+    private Map<String, String> ligatures;
+
+    private String ligaturelessContent;
+
+    /**
+     * Contains position mapping for characters between ligaturelessContent and stringContent
+     */
+    private BidiMap<Integer, Integer> ligatureStringPositionMap;
```

Comment:
Is there an entry for every character in this map?

Reply:
Yes, see the answer to line 45/48